services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from datetime import datetime
+from db.models import Order, Ticket, User
+from db.models import MovieSession
+
+
+def create_order(tickets: list, username: str, date: str = None) -> User:
+    try:
+        user = User.objects.get(username=username)
+    except User.DoesNotExist:
+        raise ValueError("No User matches the given query.")
+
+    order_date = datetime.fromisoformat(date) if date else datetime.now()
+
+    with transaction.atomic():
+        order = Order.objects.create(user=user, created_at=order_date)
+
+        for ticket_data in tickets:
+            movie_session_id = ticket_data.pop("movie_session")
+            movie_session = MovieSession.objects.get(id=movie_session_id)
```

Comment:
The `get` method can raise a `DoesNotExist` exception if no `MovieSession` with the given ID is found. Consider handling this exception to prevent runtime errors.